src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -282,11 +274,11 @@ ImageReplayer<I>::ImageReplayer(Threads *threads,
   // re-registered using "remote_pool_name/remote_image_name" name.
 
   std::string pool_name;
-  int r = m_remote->pool_reverse_lookup(m_remote_pool_id, &pool_name);
+  int r = m_local->pool_reverse_lookup(m_local_pool_id, &pool_name);
   if (r < 0) {
-    derr << "error resolving remote pool " << m_remote_pool_id
+    derr << "error resolving remote pool " << m_local_pool_id
```

Comment:
@dillaman I think this error message is confusing now -- I suppose users would thinking the ID in the message is from the remote cluster.